packages/relay/src/ethers/signer.test.ts

```diff
@@ -64,6 +64,7 @@ describe('ethers/signer', () => {
     provider._wrapTransaction.mockImplementation((arg) => ({
       ...tx,
       ...arg,
+      gasPrice: BigNumber.from(tx.gasPrice || arg.gasPrice),
```

Comment:
this line and extra `case ` a few files down seem slightly out of place, so just want to call them out, but I don't have an issue

Reply:
Thanks for noticing, but these are alright. Actually they had to be made due to the dependency upgrade and makes more sense to have them here rather than in #127